### PR TITLE
replace USE_500HZ with unlisted USE_2400GHZ_25HZ

### PR DESCRIPTION
**Pull request by JyeSmith**
With opentx nightly now supporting 500Hz this should become the default.  Users wanting stupid LR on 2.4G can use the proposed unlisted def USE_2400GHZ_25HZ.

### CODE REVIEW

File: src/src/common.cpp [hunk 1]
```diff
@@ -23,19 +23,7 @@ expresslrs_rf_pref_params_s ExpressLRS_AirRateRFperf[RATE_MAX] = {
...
-    {2, RATE_150HZ, SX1280_LORA_BW_0800, SX1280_LORA_SF7, SX1280_LORA_CR_LI_4_7, 6666, TLM_RATIO_1_32, 4, 12},
-    {3, RATE_50HZ, SX1280_LORA_BW_0800, SX1280_LORA_SF9, SX1280_LORA_CR_LI_4_6, 20000, TLM_RATIO_NO_TLM, 4, 12}};
-
-expresslrs_rf_pref_params_s ExpressLRS_AirRateRFperf[RATE_MAX] = {
-    {0, RATE_500HZ, -105, 4380, 3500, 1000, 2000, 5000},
-    {1, RATE_250HZ, -108, 4380, 3500, 2500, 2000, 5000},
-    {2, RATE_150HZ, -112, 8770, 3500, 2500, 2000, 5000},
-    {3, RATE_50HZ, -117, 17540, 3500, 2500, 2000, 5000}};
-#else
+#ifdef USE_2400GHZ_25HZ
```
Comment (cruwaller) [inline src/src/common.cpp:26]: Is the LUA script aligned with this?

**Review by cruwaller — APPROVED** [review]

**Comment by 0crap** [comment]
> With opentx nightly now supporting 500Hz this should become the default. Users wanting stupid LR on 2.4G can use the proposed unlisted def USE_2400GHZ_25HZ.

Does OpenTX scheduler nightly allow for 25Hz?
If not just remove it completely?

File: src/src/common.cpp (hunk 1 above)
Comment (pkendall64) [reply src/src/common.cpp:26]: erm 2400MHZ 😄  not GHZ

Comment (JyeSmith) [reply src/src/common.cpp]: Speeeeeed

**Comment by JyeSmith** [comment]
@0crap it does but the mixer wont go below 40Hz.  So OpenTX devs recommend setting mixer to 50Hz and only using every second packets.  But in reality it doesn't matter at these low rate for LR.

**Comment by jurgelenas** [comment]
@JyeSmith if `USE_2400MHZ_25HZ` is not not at least in user_defines.txt comments it will not be visible in the configurator.

**Comment by JyeSmith** [comment]
@jurgelenas Im open to adding it... actually we better since 25Hz is promoted on the readme.

**Comment by jurgelenas** [comment]
> @jurgelenas Im open to adding it... actually we better since 25Hz is promoted on the readme.

All good now.

**Pull request closed without merging**